sky/server/requests/executor.py

```diff
@@ -93,21 +94,21 @@ def __init__(self,
         else:
             raise RuntimeError(f'Invalid queue backend: {backend}')
 
-    def put(self, request: Tuple[str, bool]) -> None:
+    def put(self, request: Tuple[str, bool, bool]) -> None:
         """Put and request to the queue.
 
         Args:
-            request: A tuple of request_id and ignore_return_value.
+            request: A tuple of request_id, ignore_return_value, and retryable.
```

Comment:
We should probably create a dataclass or namedtuple for this at some point. Doesn't block this PR.

Reply:
Yeah, sounds good!